sync2jira/downstream_issue.py

```diff
@@ -646,7 +639,11 @@ def _create_jira_issue(client, issue, config):
     if 'labels' in issue.downstream.keys():
         kwargs['labels'] = issue.downstream['labels']
 
-    log.info("Creating issue.")
+    log.info("Creating issue for %r:  %r", issue, kwargs)
```

Comment:
This is not really a storypoints PR related change. Since it's a very small change to improve logging, I am okay with this change.